inc/saiobject.h

```diff
@@ -249,6 +249,56 @@ sai_status_t sai_query_stats_capability(
         _In_ sai_object_type_t object_type,
         _Inout_ sai_stat_capability_list_t *stats_capability);
 
+/**
+ * @brief Bulk objects get statistics.
+ *
+ * @param[in] switch_id SAI Switch object id
+ * @param[in] object_type Object type
+ * @param[in] object_count Number of objects to get the stats
+ * @param[in] object_key List of object keys
+ * @param[in] number_of_counters Number of counters in the array
+ * @param[in] counter_ids Specifies the array of counter ids
+ * @param[in] mode Statistics mode
+ * @param[out] status Array of status for each object. Length of the array should be object_count. Should be looked only if API return is not SAI_STATUS_SUCCESS.
```

Comment:
will be a bit more consistent to call it
_Inout_ sai_status_t *object_statuses
same as in sai_bulk_get_attribute

Also it is in, out, as allocated by caller, and filled by function

Reply:
> will be a bit more consistent to call it _Inout_ sai_status_t *object_statuses same as in sai_bulk_get_attribute
> 
> Also it is in, out, as allocated by caller, and filled by function

Changed the parameter name as well as inout param.
Regarding inout I see some inconsistency. Check saiswig.h for object_statuses

https://github.com/opencomputeproject/SAI/pull/1352/commits/2b409fb5830a3f5b60485add8a19f81173a05b30